### ISSUE TITLE
Right Click Menu for Windows-Terminal in Chinese Environment

### ISSUE THREAD

**Issue by LittleNewton** [microsoft/terminal#10835]
Windows Terminal Version: 1.9.1942.0
Language Environment: Simplified Chinese

Right Click menu for Windows Terminal shows like 

```
在 Windows 终端 中打开
```

This is very ugly in Chinese environment. We never add a SPACE char between two Chinese characters. I wish it would look like the following:

```
在 Windows 终端中打开    // no more SPACE char
在 Windows 终端中以管理员身份打开   // Administrator
```

**If Terminal can be opened as administrator in right-click menu, it could be more useful.**

**Comment by skyline75489** [comment]
I don’t think this is possible, considering how localization works in Windows…

**Comment by LittleNewton** [comment]
![image](https://user-images.githubusercontent.com/17793356/127745151-72e47943-c531-4e82-843d-ca4643fe0814.png)

I have realize this feature. However it should be integrated into the main dev-line.

Localization is important to a organization who want to survive in China market. @skyline75489

**Comment by lhecker** [comment]
As far as I can see the context menu entry is now translated as "在终端中打开" (stable version) and "在终端预览中打开" (preview version) respectively. This fix should be first released in 1.12.10393.0 later this week.
